python-sdk/pachyderm_sdk/api/__init__.py

```diff
@@ -0,0 +1 @@
+"""Python bindings and API stubs for the Pachyderm public API."""
```

Comment:
Is this change an accidental inclusion of the effects of testing the below `entrypoint.sh` addition?

Reply:
Not accidental. The documentation generation tool was throwing a warning because this file was empty. Rather than suppress that warning I decided to just add a docstring so this file isn't empty.